neurodamus/connection_manager.py

```diff
@@ -9,6 +9,7 @@
 from itertools import chain
 from os import path as ospath
 from typing import List, Optional
+from libsonata._libsonata import SonataError
```

Comment:
importing from private (ie: leading underscore) parts of packages is not generally recommended.

The following should work:
`from libsonata import SonataError`